client/src/components/signing-form/signing-form.js

```diff
@@ -0,0 +1,47 @@
+import ko from 'knockout';
+
+import 'components/signing-form/signing-form.less';
+import template from 'components/signing-form/signing-form.html';
+import register from 'components/component-registrator';
+
+import {user} from 'providers/user-provider';
```

Comment:
Again, name `user` doesn't really correspond to the module that's being required.